### PR TITLE
[not for merge] Testing node for pairing tmrw 

### PR DESCRIPTION
**Pull request by amyrlam**
- [ ] See if https://github.com/hashicorp/flight/issues/144 is sorted 
- [ ] #spritegate https://github.com/hashicorp/flight/issues/198 (not certain if this is a 1.0) 
- [ ] https://github.com/hashicorp/flight/issues/200 from DM, sounds like didoo will change the script and not use inline style. confirm at meeting 
- [ ] https://github.com/hashicorp/flight/issues/77 seems like can be closed

### CODE REVIEW

File: flight-icons/scripts/sync.ts [hunk 1]
```diff
@@ -37,6 +37,16 @@ import { config } from './config';
 async function sync() {
 
     // remove existing output folder
+    /* 
+        Question: do we need to run this every time, or could we just overwrite icons that have changed? 
+        No strong opinion, just curious of the thought here.
```
Comment (didoo) [inline flight-icons/scripts/sync.ts:42]: Icons can also be removed, or renamed, not just added (rare, but it happens). That's why it's easier to just delete the output folder and don't have special logic to handle that.

File: flight-icons/scripts/sync.ts [hunk 2]
```diff
@@ -37,6 +37,16 @@ import { config } from './config';
 async function sync() {
 
     // remove existing output folder
+    /* 
+        Question: do we need to run this every time, or could we just overwrite icons that have changed? 
+        No strong opinion, just curious of the thought here. 
+        I guess the catalog.json has the date, so the corresponding SVGs should also be fresh.
+        
+        - Is a good next step to have ember-flight-icons consume flight-icons? 
+        - I'm not really following flight-icons-svg vs. flight-icons-svg-sprite. Those could also be packages if we want everything atomic?
```
Comment (didoo) [inline flight-icons/scripts/sync.ts:46]: Let's discuss together, I can explain the thinking (and what options I see).

**Review by didoo — COMMENTED** [review]
Added a couple of comments

**Comment by amyrlam** [comment]
this all makes sense from us pairing today, was just testing in public, closing!

**Pull request closed without merging**